api/nakadi-event-bus-api.yaml

```diff
@@ -474,7 +859,12 @@ definitions:
         type: string
       offset:
         type: string
-  SimpleStreamEvent:
+  EventStreamBatch:
+    description: |
+      One chunk of events in a stream. A batch consists of an array of `Event`s plus a `Cursor` pointing to
+      the offset of the last Event in the stream.
```

Comment:
To the offset of an event to be delivered directly after the last event in `EventStreamBatch.events` array?

Do I remember this correctly? Was the semantic inclusive or exclusive? We discussed it several times already and several times we decided on this already :D